chat-components/ui/src/main/java/com/pubnub/components/chat/ui/component/message/renderer/GroupMessageRenderer.kt

```diff
@@ -243,7 +243,7 @@ object GroupMessageRenderer : MessageRenderer {
                                 ChatText(
                                     message = message,
                                     theme = theme.text,
-                                    placeholder = theme.text.modifier.then(
+                                    placeholderModifier = theme.text.modifier.then(
```

Comment:
@kazydek `placeholder` renamed to `placeholderModifier`

Reply:
@Chesteer89 Thx, it wasn't mentioned in the docs so no update was required.